package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@raisinten/aes-crypto-js",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
The version changes in this file and package.json are not necessary for this PR, so these can be dropped